server.js

```diff
@@ -128,23 +139,22 @@ app.use(express.json());
 //--------------------------------------------
 //CONTROLLERS
 //ADD ISAUTHENTICATED HERE - AFTER FE CONDITIONAL ADDED
-app.use("/goals", require("./Controllers/goals"));
+app.use("/goals", isAuthenticated, require("./Controllers/goals"));
```

Comment:
Did you test that this works with front end? I don't think we have front end logic to support this yet. I was planning on waiting to add this back until the front end logic is complete.